_drafts/2020-01-01-nyu-cs-transfer-perspective.md

```diff
@@ -4,45 +4,37 @@ date: 2020-01-01 6:30:00 -0700
-programming skills and the recruiting scene is decent. I agree. But here's a few reasons why I left.
+programming skills and the recruiting scene is decent. I agree. But here're a few reasons why I left.
 
 # Class Size
 
-There are 250 kids in my introductory courses. They are weed-out classes so the size makes sense. However,
-it left less room for help from TA and professor. Usually, there's a long line to get help on projects. Luckily,
-I usually came super early to avoid the crowds and am able to get help on difficult projects. Those who came 30 minutes later are not fortunate. Additionally, it leaves less room to ask questions during classes as the professor needs to go through lectures with
-250 people in the lecture hall.
+There were 250 kids in my introductory courses. They were weed-out classes so the size made sense. However,
+there was usually a long line to get help on projects during office hour. As a new programmer, these time slots were crucial to
+understand the assignments and reinforces the concepts in lectures. I usually came early to avoid the crowd and was able to get guidance on difficult projects. Those who came 30 minutes later were not so fortunate. With such big class size, it left fewer opportunities to ask questions during class since the professors need to go through lectures in a timely manner.
 
 # Waitlisting
 
-This is a common problem with NYU CS as seen by Data Management and AIT with 120 on the waitlist. However, it is also not a
-foreign [problem](https://dbknews.com/2017/12/11/umd-computer-science-petition-waitlists-iribe-differential-tuition-class-size/) to UMD.
-I suspect this is also commmon to other institutions recently as there is a rise in popularity of CS major. So it's not avoidable especially
-with the more popular course or professor. I recall being on the waitlist for introductory course until the last day of the break before getting into it.
+This is a problem with NYU CS as seen by Data Management and AIT with 120 on the waitlist by the end of the registration process for Spring 2021. However, it is also not a foreign [problem](https://dbknews.com/2017/12/11/umd-computer-science-petition-waitlists-iribe-differential-tuition-class-size/) to UMD. I suspect this also happens at other institutions recently as there is an increase in number of students majoring in CS. So it's unavoidable, especially with the more popular courses or professors. I recalled being on the waitlist for introductory courses at UMD until the last day of add/drop period before getting into it. Solving waitlisting is difficult as the demand for CS courses keeps going up but universities don't quite have the resources to hire more qualified faculty. One possible solution specific to NYU CS was outlined [here](https://blog.torchnyu.com/2020/12/08/waitlisted-professor-allocation.html) but I'm not sure if it's can be done anytime soon.
```

Comment:
I'd specify what AIT is, as not everybody will know

Reply:
You could probably omit the recently:

> I suspect this also happens at other institutions, as there is an increase in the number of students majoring in CS. So it's unavoidable, especially with the more popular courses or professors.

Reply:
"I recall" not "I recalled" 😄 . You are currently recalling, you didn't recall in the past

Reply:
"before getting into it" -> "before getting in" or if you prefer "before getting a spot"

Reply:
"if it's can be done anytime soon" -> "if it can be done any time soon"